src/multi-level-checkbox-editor/node-box.tsx

```diff
@@ -94,7 +94,19 @@ const NodeBox: React.FunctionComponent<Props> = (props) => (
                                 value={getNodeState(node)}
                                 onChange={props.setNodeValue.bind(null, node)}
                             />
-                            {node.label}
+                            <div className={'node-label'}>
+                                {node.label.split('\n').map((label) => (
+                                    <div>
+                                        {label.includes('\u2063—') &&
```

Comment:
I think this is pretty hacky. What is the purpose?

All components in this repo should be entirely agnostic of any domain/business knowledge. AccountName is falling into that category.

If you would like to have further explanation, just let me know.

Reply:
Completely agreed that it should be agnostic and even I do not like these changes but currently, we get the data from the backend in this format 
id: "process_exits_10"
label: "Checkout alt - Basket/Login/CO Process Exit Steps ⁣— INT Esprit AT↵Kaufprozess Process Exit Steps ⁣— INT Knigge Campaigns v2↵(These elements derive from the same ID)"
type: "DIMENSION"
value: false
children: []

which will be changed later since changing now will require lot of efforts from the backend and front end since we expect to live very soon.

Reply:
will remove this changes for now we will do it as part of next iteration